python/ray/data/impl/null_aggregate.py

```diff
@@ -135,52 +183,35 @@ def _accum(a: AggType, r: T) -> AggType:
     return _accum
 
 
-def _null_wrap_merge(
+def _null_wrap_vectorized_aggregate(
     ignore_nulls: bool,
-    merge: Callable[[AggType, AggType], AggType],
-) -> AggType:
+    vec_agg: Callable[[BlockAccessor[T]], AggType],
+) -> Callable[[BlockAccessor[T]], AggType]:
```

Comment:
The return type should be `Callable[[AggType, Block[T]], AggType]`? I think it doesn't match the previous type annotation.

Reply:
Ah yep this is a holdover from a previous implementation, I'll fix it!